samples/modules/nanopb/src/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2011 Petteri Aimonen
+ * Copyright (c) 2021 Basalte bv
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+
+#include <pb_encode.h>
+#include <pb_decode.h>
+#include "src/simple.pb.h"
+
+int main(void)
+{
+	/* This is the buffer where we will store our message. */
+	uint8_t buffer[SimpleMessage_size];
+	size_t message_length;
+	bool status;
+
+	/* Encode our message */
+	{
```

Comment:
Can you please remove these scope brackets, and the ones below? This is not something we usually make use of.

Reply:
I've moved the scoped logic to functions, this is to demonstrate the stack usage of the message struct.